### PR TITLE
feat(playwright)/new api: `.clear()`, `.blur()`, `.focus()`

### PR DESCRIPTION
**Pull request by EgorBodnar**
### The reason for the PR is the Integration of the new Playwright API into CodeceptJS and the bumping Playwright version to `1.32.3`.

 **A list of the new Playwright API, which is important to the CodeceptJS framework, but has not yet been included**:
* Added in [v1.28 of Playwright](https://playwright.dev/docs/release-notes#version-128)
``` js
.blur()
.clear()
```
* Added in [v1.14 of Playwright](https://playwright.dev/docs/release-notes#version-114)
``` js
.focus()
```

 ### As part of this PR added implementations of these methods in Codeceptjs.
- Bringing the name of options to a general form: [opts vs options](https://github.com/EgorBodnar/CodeceptJS/commit/698701c71679c92a8240b4e6d79bea791a0ad485)

 - Added tests, HTML forms for tests and documentation:
    * `I.focus()`
    * `I.blur()`
    * `I.clearField()`modified  | old `I.clearField()` In fact, didn't clean the field, but replaced its value with an empty string. It doesn't always work. So it's better to use the new Playwright API (`.clear`) for this.  Backward compatibility is preserved for those who use the old version of the Playwright. 

- Created feature request #3677 because all the new Playwright functionality is tied to the [Locator](https://playwright.dev/docs/api/class-locator) and it is impossible to use the new Playwright API for elements found through the [ElementHandle](https://playwright.dev/docs/api/class-elementhandle#element-handle-query-selector-all) only if it does not match with `devtools-protocol` API.

- Added temporary solution function [getXPathForElement(elementHandle)](https://github.com/codeceptjs/CodeceptJS/pull/3665/commits/c4efd65f6dfb281d30650ee1b27b332f7f22c85b) to use like a bridge between old [ElementHandle](https://playwright.dev/docs/api/class-elementhandle#element-handle-query-selector-all) and new [Locator](https://playwright.dev/docs/api/class-locator)

Applicable helpers:

- [X] Playwright

## Type of change

- [X] :rocket: New functionality
- [X] :nail_care: Polish code

## Checklist:

- [X] Tests have been added
- [X] Documentation has been added (Run `npm run docs`)
- [X] Lint checking (Run `npm run lint`)
- [X] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

**Review by kobenguyent — APPROVED** [review]

File: test/helper/Playwright_test.js
Comment (kobenguyent) [inline test/helper/Playwright_test.js]: Just curious if we could use await/async here?

Comment (EgorBodnar) [reply test/helper/Playwright_test.js]: Agree with you. Fixed

File: lib/helper/Playwright.js
Comment (EgorBodnar) [inline lib/helper/Playwright.js]: @DavertMik what is the proper way to locate the element here? 
use `._locate()` or `.findClickable()`? 
Because `.findClickable()` in fact, not just looking for an available element to click, but just well and steadily locating an element

File: lib/helper/Playwright.js
Comment (peterngtr) [inline lib/helper/Playwright.js]: What do you think if we name it clearFieldValue()

Comment (EgorBodnar) [reply lib/helper/Playwright.js]: In a good way, we need to completely replace the `.clearField()`. 
But the`.clearField()` has already been used many times by users. 

#### Here are the possible options, as I see it:
1.  we need to either replace it slowly in the next Codeceptjs **major** version,
2. leave it as it is and remove deprecated in the next versions
3. completely replace the `.clearField()` here

So we need a final decision from @DavertMik

Comment (kobenguyent) [reply lib/helper/Playwright.js]: I'm up with replacing the implementation of clearField() instead of introducing different names. As far as I know the current clearField still brings some issues.

Comment (EgorBodnar) [reply lib/helper/Playwright.js]: I've the same opinion as you.
As this is a major change, need approval from @DavertMik

Comment (DavertMik) [reply lib/helper/Playwright.js]: > I'm up with replacing the implementation of clearField() instead of introducing different names. As far as I know the current clearField still brings some issues.

@EgorBodnar yes, I agree to replace implementation

It's not a major change as the behavior should be the same as current clearField
Sure, implementation is different and may bring some edge cases issue but I agree, no need to introduce new method

File: lib/helper/Playwright.js
Comment (DavertMik) [reply lib/helper/Playwright.js]: I think `_locate` is better here

**Review by DavertMik — CHANGES_REQUESTED** [review]
Here is the thing. Playwright introduced `clear` only in recent releases, while we had clearField (not with the best implementation) from the very beginning :smiley: 

But now please replace `clearField` implementation with a native one and I'm merging

File: lib/helper/Playwright.js
Comment (EgorBodnar) [reply lib/helper/Playwright.js]: replaced

**Review by DavertMik — APPROVED** [review]

**Comment by DavertMik** [comment]
Thank you so much!

**Pull request merged**
